lib/MusicBrainz/Server/Data/Artist.pm

```diff
@@ -247,7 +247,7 @@ sub delete
     $self->rating->delete(@artist_ids);
     $self->subscription->delete(@artist_ids);
     $self->remove_gid_redirects(@artist_ids);
-    $self->delete_returning_gids('artist', @artist_ids);
+    $self->delete_returning_gids(@artist_ids);
```

Comment:
The function name is admittedly a bit strange since we aren’t actually using the returned MBIDs, but there is one place where they are used, and I wasn’t in a mood to change the name.